ush/python/pygfs/ufswm/ufs.py

```diff
@@ -8,6 +8,7 @@
 from pygw.template import Template, TemplateConstants
 from pygw.file_utils import FileHandler
 from pygw.logger import logit
+from pygw.timetools import datetime_to_YMDH
```

Comment:
Don't need this if you can access the year, month, day and hour directly from the datetime object.